Library/Homebrew/global.rb

```diff
@@ -44,7 +45,8 @@
 # Bintray fallback is here for people auto-updating from a version where
 # HOMEBREW_BOTTLE_DEFAULT_DOMAIN isn't set.
 HOMEBREW_BOTTLE_DEFAULT_DOMAIN = ENV["HOMEBREW_BOTTLE_DEFAULT_DOMAIN"] ||
-                                 "https://homebrew.bintray.com"
+                                 "https://#{OS.mac? ? "homebrew" : "linuxbrew"}.bintray.com"
```

Comment:
Can you use an `if`/`else` here instead? I'm also not entirely sure this is necessary as it's a fallback from when you're updating from and old Homebrew version. I've been wondering for a while if there's a way to resource `brew.sh` in that case (but maybe that's a separate issue)?